addons/backpacks/CfgVehicles.hpp

```diff
@@ -84,6 +84,16 @@ class CfgVehicles {
         hiddenSelectionsTextures[] = {"\a3\weapons_f\ammoboxes\bags\data\backpack_tortila_blk_co.paa"};
     };
 
+    class CLASS(Backpack_Carryall_White): CLASS(Backpack_Carryall_DarkBlack) {
+        author = "GilleeDoo, Mike";
```

Comment:
```suggestion
    class CLASS(Backpack_Carryall_White): B_Carryall_Base {
        dlc = QUOTE(PREFIX);
        scope = 2;
        author = "GilleeDoo, Mike";
```

Reply:
Why ignore inheritance?

Reply:
It's 2 entries, inheritance or not doesn't make a difference. Everything else in the file is not inheriting in a nested way, so this shouldn't either.